.github/CONTRIBUTING.md

```diff
@@ -31,70 +31,93 @@ If you at any point have any questions please read: [I have a question related t
+    - Move the issue to the "Ready to do"-column in the "Kirby kan-ban board"-project.
+3. **Implementation:**
+    - Make sure you have read: [before you get coding](#before-you-get-coding).
+    - Signal to others you are working on the issue by assigning yourself.
+    - Create a "feature"-branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master). 
+    - Publish a WIP implementation to Github as a draft PR and ask [#kirby-guild](#getting-access-to-the-kirby-guild-slack-channel) for feedback. <!--This will let you know early on if you are going in the right direction.-->
+    - Make sure you have implemented tests following the "[the good test](#test)"-guidelines.
+    - Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.
+4. **Review:**
+    - Do a [self-review](#self-review). 
+    - Create a pull-request (see [creating pull-requests](#creating-pull-requests)).
+    - Request that the pull-request is code-reviewed in [#kirby-guild](#getting-access-to-the-kirby-guild-slack-channel).
+    - Request that the pull-request is [UX reviewed](#ux-review).
+    - When the pull-request has been approved it will be automatically merged to master via [automerge](https://docs.github.com/en/github/collaborating-with-issues-and-pull-requests/automatically-merging-a-pull-request).
+5. **Celebrate! 🎉**
+
+### Checklist: I have an enhancement 
+If you have an idea for Kirby that is not a brand new component or aimed at improving code quality; then you should submit an enhancement request!
+The process is the same as submitting a component request just using another issue-template: 
+
+1. **Open a [new issue](https://github.com/kirbydesign/designsystem/issues/new/choose) using the "enhancement request"-template** (See guidelines on how to write: "[the good issue](#issue)").
+2. **Follow steps 2 to 5 in "[I have a component request](#i-have-a-component-request)".**
+
+### Checklist: I would like to report a bug 
+If you have found a bug in Kirby you should follow these steps: 
+1. **Report it by opening a [new issue](https://github.com/kirbydesign/designsystem/issues/new/choose) using the "bug report"-template** (See guidelines on how to write [the good issue](#issue)).
+2. **Get the bug verified** 
+    - To make sure the bug is not intended behaviour contact [#kirby-guild](#getting-access-to-the-kirby-guild-slack-channel) to have it verified; do not proceed until this is done. 
+3. **Implementation:**: 
+    - Signal to others you are working on the issue by assigning yourself.
+    - Create a "fix"-branch from the [master branch](https://github.com/kirbydesign/designsystem/tree/master). 
+    - Create a test that reproduces the bug following "[the good test](#test)"-guidelines. 
+    - Write code to fix the bug
+5. **Follow steps 4 and 5 in "[I have a component request](#i-have-a-component-request)".**
```

Comment:
I don't think we have had this discussion, but I think it would be a good idea with some sort of "post mortem" for bugs - depending on what the bug and the fix is, of course.

Should we add another step for this?

Reply:
Sound reasonable - perhaps we should have a talk about it in the team about how to do this the best, and add it to the contribution guidelines afterwards?